### DISCUSSION TITLE
Critical error + JS Synthax error

### DISCUSSION THREAD

**Discussion by julien-paris** [code4recovery/12-step-meeting-list#616] [General]
Hi folks and happy holidays,
I have 2 issues:
1. All updates after 3.11.3 create a critical error on the site
2. We use various sources to import meetings and this one now stopped working:
https://aa-israel.org/wp-admin/admin-ajax.php?action=meetings
Error: JSON: Syntax error, malformed JSON.

Please advise,
Much thanks!
Julien

**Comment by kiyote33** [comment]
#Response # 
Hi Julien,

Thanks for reporting the issue you are having and providing the link to your json feed. I've done some investigating and here's what I've found:
1. I was able to replicate your issue in my test box.
2. I ran your json through a couple different validators and they all passed.
3. I then copied the json into a text file and loaded it locally. It successfully loaded the 12 Step Meeting List with 115 records.
4. A deeper dive into the import process with the link you provided discovered that the import was failing because the SSL certificate verification is turned off on your server. To be more precise, this is the 403 Forbidden debug message returned: **Certificate verification disabled (sslverify=false) https://aa-israel.org/wp-admin/admin-ajax.php?action=meetings**

I believe you simply need to get the aa-israel.org hosting provider to enable ssl verification, which appears to be turned off for some reason.

I hope this helps you resolve the issue on your end.
